library/include/ck/library/host_tensor/host_tensor.hpp

```diff
@@ -341,19 +327,20 @@ float check_error(const Tensor<T>& ref, const Tensor<T>& result)
     {
         for(int i = 0; i < ref.mData.size(); ++i)
         {
-            error += std::abs(double(ref.mData[i]) - double(result.mData[i]));
-            float diff = std::abs(double(ref.mData[i]) - double(result.mData[i]));
+            error += std::abs(float(ref.mData[i]) - float(result.mData[i]));
```

Comment:
should we rename `error` and `max_diff` to `l1_err` and `linf_err`?

Reply:
fixed